docs/source/java/vector_schema_root.rst

```diff
@@ -109,7 +109,7 @@ Here is the example of building a :class:`VectorSchemaRoot`
 
 The vectors within a :class:`VectorSchemaRoot` could be loaded/unloaded via :class:`VectorLoader` and :class:`VectorUnloader`.
 :class:`VectorLoader` and :class:`VectorUnloader` handles converting between :class:`VectorSchemaRoot` and :class:`ArrowRecordBatch` (
-representation of a RecordBatch :doc:`IPC <../format/IPC.rst>` message). Examples as below
+representation of a RecordBatch :doc:`IPC <../format/IPC>` message). Examples as below
```

Comment:
This file is no longer containing the actual contents, so pointing directly to the section in Columnar.rst might be better: `` :ref:`IPC <format-ipc>` ``

Reply:
Thanks, I gave this section a quick pass and did some copyediting in general